internal/handlers/registration_handler.go

```diff
@@ -77,6 +78,24 @@ func RegistrationListHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func RegistrationCreateHandler(w http.ResponseWriter, r *http.Request) {
+	id := identity.Get(r.Context())
+	db := store.GetStore()
+
+	if config.Get().AllowlistEnabled {
+		allowed, err := db.AllowedIP(&store.Address{
+			IP:    r.Header.Get("x-forwarded-for"),
```

Comment:
How about we make this header configurable ( with x-forwarded-for) being the default?